Sources/SQLiteKit/SQLiteConnection+SQLKit.swift

```diff
@@ -82,7 +82,7 @@ internal struct _SQLiteDatabaseVersion: SQLDatabaseReportedVersion {
 }
 
 private struct _SQLiteSQLDatabase: SQLDatabase {
-    let database: SQLiteDatabase
+    let database: any SQLiteDatabase
```

Comment:
FYI they're dropping the ExistentialAny upcoming feature. I personally liked it. Not sure what exactly are their plans.

Reply:
Citation or it didn't happen 😜 What an absurd thing to do...